src/platform/ESP32/Logging.cpp

```diff
@@ -26,35 +26,20 @@ void ENFORCE_FORMAT(3, 0) LogV(const char * module, uint8_t category, const char
     snprintf(tag, sizeof(tag), "chip[%s]", module);
     tag[sizeof(tag) - 1] = 0;
 
+    char formattedMsg[CHIP_CONFIG_LOG_MESSAGE_MAX_SIZE];
+    vsnprintf(formattedMsg, sizeof(formattedMsg), msg, v);
+
     switch (category)
     {
     case kLogCategory_Error:
-        if (esp_log_default_level >= ESP_LOG_ERROR)
-        {
-            printf(LOG_COLOR_E "E");                        // set color
-            printf(" (%u) %s: ", esp_log_timestamp(), tag); // add timestamp
-            esp_log_writev(ESP_LOG_ERROR, tag, msg, v);
-            printf(LOG_RESET_COLOR "\n");
-        }
+        ESP_LOGE(tag, "%s", formattedMsg);
```

Comment:
revert seems reasonable if this broke functionality, however the original PR seemed to have wanted to have some specific changes that we lose here.

Maybe open an issue to see what the intent here is? `ESP_LOG*` macros do look better than printf combined with esp_log_writev combined, but unsure what the original intent is.

Also `esp_log_default_level` changes seemed reasonable.

Reply:
If I am understanding this correctly, `src/platform/ESP32/Logging.cpp` is for matter logging. It will suppress log levels at compile time. Matter logs themselves have the capability of suppressing logs at runtime. I don't think we want want to have logs potential also suppressed at the platform level here

Reply:
That is a fair point ... several toggles for the same thing at different levels does sound odd.